### ISSUE TITLE
Speed replay

### ISSUE THREAD

**Issue by jetpack001** [RMPR/atbswp#37]
## Checklist

- [x ] I use the latest release of atbswp
- [x ] The issue is not existing yet

## Verbose log

```
PASTE VERBOSE LOG HERE
```

### Operating System
Ubuntu 18.04

### Desktop Environment/Window Manager
Gnome 3.28

### Python version
Write here

## Description

Everything works fine but i can't find out how to replay the capture at the same speed as i record, not faster.

**Comment by RMPR** [comment]
Can give more details? What do you feel is faster: the mouse or the keyboard?

**Comment by jetpack001** [comment]
mouse, i use it for a game and every click has to be precise at a certain time but when it clicks its too fast and rutine fails

**Comment by RMPR** [comment]
Turns out the granularity is something I defined as a constant in the code. in the next update I'll expose that setting to users. If you don't want to wait, you can modify in the source code directly. Relevant line https://github.com/RMPR/atbswp/blob/97be7406040eec44607dbea2d2518e4fb0e20562/atbswp/control.py#L121

**Comment by jetpack001** [comment]
find the line but i don't know what value should i put to be fine, kinda newbie into that.

**Comment by RMPR** [comment]
To be honest, me neither. I guess you can find the value that suits you
the most by experimenting. That's also why I want to expose this setting
to end-users.

**Comment by jetpack001** [comment]
ok, ill try. thank you

**Comment by whoyawn** [comment]
Piggybacking off this, where would be the best place to write delays between key presses? I'm also trying to replicate exact timings for games.

**Comment by RMPR** [comment]
If I understand correctly you have a delay problem but with the keyboard
instead of the mouse?

**Comment by whoyawn** [comment]
Yes, that is correct. Currently, each keypress has a consistent delay in between and I'm trying to find a way to record the exact time in between them.

**Comment by RMPR** [comment]
Okay, please create a new issue for this, feel free to add any other detail you might find relevant.

**Comment by paomontes** [comment]
Hello RMPR,
First excellent work, 

Second i have a question, 
I want to make the playback faster, the mouse and keyboard, can you suggest me how to do it? 

I undestand that in control play i have to change the value of the next line to have a faster mouse playback and the keyboard?

control.py
Line 121 in 
 self.mouse_sensibility = 21

Thanks in advance for your advice,
Have a nice day,
Paola.

**Comment by RMPR** [comment]
> I undestand that in control play i have to change the value of the next line to have a faster mouse playback and the keyboard?
> 
> control.py Line 121 in self.mouse_sensibility = 21

increasing this value will make you have a faster mouse playback, for the keyboard it's a bit more complicated than that. If you don't want to fiddle with the code just for that, you can create an issue to expose this setting to end-users, I'll have a look when I'll have more bandwidth.

**Comment by RMPR** [comment]
Seems like a good idea, I can definitely accept a PR about this.
